xml/lxc.xml

```diff
@@ -163,6 +163,142 @@ Michal Svec (PM)
+   <callout arearefs="co.lxc.netw.addr">
+    <para>
+     Contains the IP address.
+    </para>
+   </callout>
+   <callout arearefs="co.lxc.netw.prefix">
+    <para>
+     Optional parameter (will be automatically set if not specified). Defines the
+     number of 1 bits in the netmask. For IPv4, the default prefix is determined
+     according to the network <quote>class</quote> (<literal>A</literal>,
+     <literal>B</literal>, or <literal>C</literal>). For IPv6, the default prefix
+     is <literal>64</literal>.
+    </para>
+   </callout>
+  </calloutlist>
+ </step>
+ <step>
+    <para>
+     You can also add route elements to define IP routes to add in the guest.
+     This is used by the LXC driver.
+    </para>
+  <screen>[...]
+&lt;devices>
+ &lt;interface type<co xml:id="co.lxc.netw.ethernet"/>='ethernet'>
+  &lt;source/><co xml:id="co.lxc.netw.source"/>
+   &lt;ip address<co xml:id="co.lxc.netw.ip1"/>='192.168.123.1' prefix='24'/>
+   &lt;ip address<co xml:id="co.lxc.netw.ip2"/>='10.0.0.10' prefix='24' peer='192.168.122.5'/>
+   &lt;route<co xml:id="co.lxc.netw.route"/> family='ipv4' address='192.168.42.0' prefix='24'
+            gateway='192.168.123.4'/>
+  &lt;source/>
+  [...]
+ &lt;/interface>
+ [...]
+&lt;/devices>
+[...]</screen>
+  <calloutlist>
+   <callout arearefs="co.lxc.netw.ethernet">
+      <para>
+       Network devices of type <literal>ethernet</literal> can optionally be
+       provided with one or multiple IP addresses (<xref linkend="co.lxc.netw.ip1"
+        xrefstyle="select:label"/>, <xref linkend="co.lxc.netw.ip2"
+        xrefstyle="select:label"/>) and with one or multiple routes (<xref
+         linkend="co.lxc.netw.route" xrefstyle="select:label"/>)
+       to set on the host side of the network device.
+      </para>
+    <para>
+     These are configured as subelements of the <literal>source</literal> element
+     (<xref linkend="co.lxc.netw.source"/>) of the interface. They have the same
+     attributes as the similarly named elements used to configure the guest side
+     of the interface (see the step above).
+    </para>
+   </callout>
+   <callout arearefs="co.lxc.netw.ip1">
+    <para>
+     First IP address for the network device of type <literal>ethernet</literal>.
+    </para>
+   </callout>
+   <callout arearefs="co.lxc.netw.ip2">
+    <para>
+     Second IP address for the network device of type <literal>ethernet</literal>.
+    </para>
+   </callout>
+   <callout arearefs="co.lxc.netw.route">
+    <para>
+     Route to set on the host side of the network device.
+    </para>
+   </callout>
+  </calloutlist>
+  <para>
+   Find further details about the attributes of this element at <link
+  xlink:href="http://libvirt.org/formatnetwork.html#elementsStaticroute"/>.
+  </para>
+ </step>
+ <step>
+  <para>
+   Save the changes and exit the editor.
+  </para>
+ </step>
+</procedure>
+  <para>
+   <remark>taroth 2017-08-25: is the following warning still correct and needed?</remark>
+  </para>
```

Comment:
The ifcfg- part of it is still true as long as bsc#883349 is not fixed.
FWICT the IPV6 part of the warning could be remove now.

Reply:
Ok, removed the second sentence and downgraded from 'Warning' to 'Note'.